pkgs/development/tools/glamoroustoolkit/default.nix

```diff
@@ -18,25 +20,39 @@
 , libglvnd
 , libuuid
 , libxcb
+, harfbuzz        # libWebView.so
+, libsoup_3       # libWebView.so
+, webkitgtk_4_1   # libWebView.so
+, zenity          # GT file selection dialog
```

Comment:
please move these comments to where the inputs are used

Reply:
Done (no other changes)